Add tokei

dbast opened the pull request.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/tokei```) and found it was in an excellent condition.

mariusvniekerk commented on recipes/tokei/meta.yaml:
```diff
@@ -0,0 +1,36 @@
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aaronepower/tokei/releases/download/v{{ version }}/tokei-v{{ version }}-x86_64-unknown-linux-gnu.tar.gz  # [linux]
```
Thanks for the submission.  
In general conda-forge prefers not to repackage binary artifacts unless there is no other way of building the package.  Could we try at least to build for osx and windows?

dbast replied:
Ok. will try that.

dbast commented:
MacOS build fails with: (signal: 11, SIGSEGV: invalid memory reference) ... searching tells, that this could be potentially related to a too old SDK... the used SDK in that build is MacOSX10.9.sdk ...

Does anybody have any idea how to resolve this?

marcelotrevisani commented:
@mariusvniekerk is that good to go?

dbast commented:
Updated the recipe:
- Now builds from source (no binary downloads, also removed the previous commit that did that)
- Builds on all 3 platforms, linux-64, mac-64 and win-64 (due to updates outside of the recipe, CI code or Rust pkg updates)
- License also packaged

@mariusvniekerk Anything else to be done here?
